src/views/public/Leaders.vue

```diff
@@ -3,7 +3,7 @@
     <div class="w-full xl:w-2/3">
       <header class="flex justify-between">
         <h5 class="text-4xl">
-          Leaders ({{leadersCount}})
+          Leaders <!-- ({{leadersCount}}) -->
```

Comment:
why comment this out?

Reply:
Honesty can't fully explain at this moment, but I plan on adding it back in the very nearest future